.changelog

```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Fixed incorrect spelling of openSUSE
```

Comment:
I'd only mention fixes in changelog that are somehow important for the reader or user.